src/components/SearchStockBlock/StockBlock.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <v-card flat>
+    <v-row
+      no-gutters
+      class="px-3"
+    >
+      <v-col
+        class="px-0 py-3"
+        cols="2"
+        sm="2"
+        md="1"
+        lg="1"
+        xl="1"
+      >
+        <v-avatar
+          class="profile"
+          size="50"
+          rounded
+          data-cy="logo"
+        >
+          <v-img :src="logo" />
+        </v-avatar>
+      </v-col>
+      <v-col
+        align-self="center"
+        cols="10"
+        sm="5"
+        md="6"
+        lg="6"
+        xl="6"
+        class="text-left mx-0 mt-1 py-0 pr-0"
+      >
+        <v-list-item color="black">
+          <v-list-item-content class="py-1">
+            <v-list-item-title
+              class="text-caption text-wrap"
+              data-cy="nasdaq"
+            >
+              NASDAQ:
```

Comment:
Would "ticker" be a more accurate label here? NASDAQ is one specific stock exchange platform, for example there's also stocks that trade on the TSX and NYSE. Unless we only display stocks that trads on NASDAQ. In which case please disregard.

Reply:
I actually didn't know this! I can change it to Ticker!